### PR TITLE
Slideshow: Autostart control

### PR DESCRIPTION
**Pull request by jeherve**
Replay of #2198

> Add autostart="false"` to [gallery] tag to pause on start.
>
> In opposite to #558 @lancewillett's statement, in my opinion it is MUST HAVE. For me, this slideshow without it is just useless.

### CODE REVIEW

**Comment by jeherve** [comment]
cc @odie2

File: modules/shortcodes/slideshow.php [hunk 1]
```diff
@@ -177,6 +178,7 @@ function shortcode_callback( $attr, $content = null ) {
 			'gallery'  => $gallery,
 			'selector' => $gallery_instance,
 			'trans'    => $attr['trans'] ? $attr['trans'] : 'fade',
+			'autostart'    => $attr['autostart'] ? $attr['autostart'] : 'true',
 			'color'    => $color,
```
Comment (roccotripaldi) [inline modules/shortcodes/slideshow.php]: Spacing is off

Comment (jeherve) [inline modules/shortcodes/slideshow.php]: Thanks, got it fixed.

**Comment by roccotripaldi** [comment]
Code looks good. Works as expected. Here is proof:
http://roccotrip.wpsandbox.me/blog/2015/06/09/gallery-that-doesnt-autostart/

**Comment by dereksmart** [comment]
:+1:

**Pull request merged**